### ISSUE TITLE
The current operating system is not capable of running this task

### ISSUE THREAD

**Issue by stevozilik** [qetza/vsts-replacetokens-task#235]
Hi,

I'm getting this error and have no idea why. I can't find anything in the documentation (or in the error message) to indicate steps to resolve this.

Would it be possible to improve the error and documentation? 

```
Starting: replacetokens
==============================================================================
Task         : Replace Tokens
Description  : Replace tokens in files
Version      : 1.4.1
Author       : Guillaume Rouchon
Help         : [More Information](https://github.com/qetza/vsts-replacetokens-task#readme)
==============================================================================
##[error]The current operating system is not capable of running this task. That typically means the task was written for Windows only. For example, written for Windows Desktop PowerShell.
Finishing: replacetokens
```

**Comment by ybinhnguyen** [comment]
Hi,

I'm also getting this error since yesterday and have no idea why. I tried 3.9.0 and 4.1.0 and it didn't work. I used version 3.8 and it didn't have problem. Can you please help? Thank you,

2021-11-24T14:43:00.3723429Z ##[section]Starting: Replace tokens in **/*.json
2021-11-24T14:43:00.3938963Z ==============================================================================
2021-11-24T14:43:00.3939315Z Task         : Replace Tokens
2021-11-24T14:43:00.3939600Z Description  : Replace tokens in files
2021-11-24T14:43:00.3939845Z Version      : 3.9.0
2021-11-24T14:43:00.3940099Z Author       : Guillaume Rouchon
2021-11-24T14:43:00.3940483Z Help         : [Learn more about this task](https://github.com/qetza/vsts-replacetokens-task/blob/master/ReplaceTokens/ReplaceTokensV3/README.md) (v3.9.0)
2021-11-24T14:43:00.3940888Z ==============================================================================
2021-11-24T14:43:00.4278817Z ##[error]A supported task execution handler was not found. The task does not carry an implementation that is compatible with your current operating system 'Windows'. Contact the task author for more details.
2021-11-24T14:43:00.4302570Z ##[section]Finishing: Replace tokens in **/*.json

2021-11-24T16:40:33.3559555Z ##[section]Starting: Replace tokens in appsettings.json
2021-11-24T16:40:33.3566023Z ==============================================================================
2021-11-24T16:40:33.3566352Z Task         : Replace Tokens
2021-11-24T16:40:33.3566607Z Description  : Replace tokens in files
2021-11-24T16:40:33.3566861Z Version      : 4.1.0
2021-11-24T16:40:33.3567146Z Author       : Guillaume Rouchon
2021-11-24T16:40:33.3567516Z Help         : [Learn more about this task](https://github.com/qetza/vsts-replacetokens-task/blob/master/ReplaceTokens/ReplaceTokensV4/README.md) (v4.1.0)
2021-11-24T16:40:33.3567913Z ==============================================================================
2021-11-24T16:40:33.3786759Z ##[error]A supported task execution handler was not found. The task does not carry an implementation that is compatible with your current operating system 'Windows'. Contact the task author for more details.
2021-11-24T16:40:33.3809701Z ##[section]Finishing: Replace tokens in appsettings.json

**Comment by qetza** [comment]
Hi @ybinhnguyen,
Yesterday i published some new versions (3.9.0 and 4.1.0) which uses the Node10 execution handler but unfortunately this only works with agent in version 2.144.0 or higher.
Today i've published new version (3.9.1 and 4.1.1) to revert to the old Node6 execution handler, please make sure that you are using one of these versions.

**Comment by ybinhnguyen** [comment]
Hello,

Thank you, it's working now with 3.9.1 and 4.1.1 version. Thank you very much for your help. Happy Thanksgiving and happy holidays!!!
